charts/psmdb-operator/README.md

```diff
@@ -0,0 +1,51 @@
+* At least `v2.4.0` version of helm
+
+## Deployment Details
+* <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>
+
+## Chart Details
+This chart will:
+* deploy a PSMDB Operator Pod for the further MongoDB creation in K8S.
+
+### Installing the Chart
+To install the chart with the `psmdb` release name using a dedicated namespace (recommended):
+
+```sh
+helm repo add percona https://percona-lab.github.io/percona-helm-charts/
+helm install my-operator percona/psmdb-operator --version 0.1.0 --namespace my-namespace
+```
+
+The chart can be customized using the following configurable parameters:
+
+| Parameter                       | Description                                                                   | Default                                   |
+| ------------------------------- | ------------------------------------------------------------------------------| ------------------------------------------|
+| `image.repository`              | PSMDB Operator Container image name                                           | `percona/percona-server-mongodb-operator` |
+| `image.tag`                     | PSMDB Operator Container image tag                                            | `1.4.0`                                   |
+| `image.pullPolicy`              | PSMDB Operator Container pull policy                                          | `Always`                                  |
+| `image.pullSecrets`             | PSMDB Operator Pod pull secret                                                | `[]`                                      |
+| `replicaCount`                  | PSMDB Operator Pod quantity                                                   | `1`                                       |
+| `tolerations`                   | List of node taints to tolerate                                               | `[]`                                      |
+| `resources`                     | Resource requests and limits                                                  | `{}`                                      |
+| `nodeSelector`                  | Labels for Pod assignment                                                     | `{}`                                      |
+| `watchNamespace`                | Set when a different from default namespace is needed to watch                | `""`                                      |
+| `nameOverride`                  | Set if the chart name is not desired to be the operators name                 | `""`                                      |
+| `fullnameOverride`              | By default operator name will contain chart name concatenated with operator name.<br>Set this variable if you want to set the whole bunch of names  | `""`                                      |
+| `createCRD`                     | Set to false if you don't want the helm chart to automatically create the CRD | `true`                                    |
```

Comment:
```suggestion
```
It looks like this option is not officially supported

Reply:
done